splash-dao-offchain/src/state_projection.rs

```diff
@@ -120,7 +121,12 @@ where
             bytes.extend_from_slice(&new_version_bytes);
             let state_key = prefixed_bytes(STATE_PREFIX, &bytes);
             let predicted = AnyMod::Predicted(entity);
-            println!("state_json: {:?}", serde_json::to_string_pretty(&predicted));
+            println!(
```

Comment:
Just want to keep the `println` here, since pretty-printing JSON is handy for debugging